### DISCUSSION TITLE
passToClient implicit error processing

### DISCUSSION THREAD

**Discussion by igolkotek** [vikejs/vike#1375] [Discussion]
According to the docs passToClient uses `@brillout/json-serializer` lib.
Also according to the docs just classes are lost during serialization.

Anyway `json-serializer` serializes OK my arrays of objects, and nested objects. But they do not appear at the client side.
And the worst it is done silently.

_Isn't it better to add some setting to make the serializer errors loud?_

Thanks

**Comment by brillout** [comment]
Indeed errors shouldn’t be swallowed. If they are then it’s a bug. Minimal reproduction welcome.

**Reply by igolkotek** [reply]
I just tried a fresh react-vike project. And it works.
How?
Gone to think about..

**Reply by igolkotek** [reply]
I put data from my app to the virgin react-vike template. And I got the data passed from server to client.
But in my project I can't (settings are the same).

Is there any hook I could use to log the passing stage?

Thanks

**Reply by brillout** [reply]
If you can't provide a reproductino then, as an alternative, I suggest digging into Vike's code, see https://vike.dev/debug.

**Reply by igolkotek** [reply]
I'll try a bit more
